### PR TITLE
Add Global mode to TimeSeriesWidget

### PR DESCRIPTION
**Pull request by Clebal**

### CODE REVIEW

File: packages/react-core/src/operations/constants/GroupDateTypes.d.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 export enum GroupDateTypes {
-  YEARS = 'years',
-  MONTHS = 'months',
-  WEEKS = 'weeks',
-  DAYS = 'days',
-  HOURS = 'hours',
-  MINUTES = 'minutes'
+  YEARS = 'year',
```
Comment (VictorVelarde) [inline packages/react-core/src/operations/constants/GroupDateTypes.d.ts:2]: Out of curiosity, why this change?

Comment (Clebal) [reply packages/react-core/src/operations/constants/GroupDateTypes.d.ts:2]: Because in all the providers, these operations are singular :(

File: packages/react-widgets/src/models/TimeSeriesModel.js [hunk 2]
```diff
@@ -1,22 +1,212 @@
...
+
+    return acc;
+  }, new Map());
+
+  return [...groups.entries()].map(([name, value]) => ({
+    name,
+    value:
+      value.filter((_, idx) => idx % 2 === 0).reduce((acc, [val]) => acc + val, 0) /
+      value.filter((_, idx) => idx % 2 !== 0).reduce((acc, [val]) => acc + val, 0)
+  }));
```
Comment (VictorVelarde) [inline packages/react-widgets/src/models/TimeSeriesModel.js:125]: This final part is a bit obscure, can you comment (or use some better variable names)?

Comment (Clebal) [reply packages/react-widgets/src/models/TimeSeriesModel.js:125]: I agree. I'm going to improve it now.

**Review by VictorVelarde — APPROVED** [review]
👍🏻
